src/test/java/picard/illumina/parser/readers/CbclReaderTest.java

```diff
@@ -31,9 +31,8 @@ public void testReadValidFile() {
         filters.put(1101, TILE_1101_FILTER);
         LocsFileReader locsFileReader = new LocsFileReader(new File("testdata/picard/illumina/readerTests/s_1_6.locs"));
         List<AbstractIlluminaPositionFileReader.PositionInfo> locs = locsFileReader.toList();
-
-        final CbclReader reader = new CbclReader(Arrays.asList(PASSING_CBCL_C99_1, PASSING_CBCL_C100_1),
-                filters, new int[]{2}, 1101, locs, false);
+        final CbclReader reader = new CbclReader(Arrays.asList(PASSING_CBCL_C1_1, PASSING_CBCL_C2_1),
+                filters, new int[]{2}, 1101, locs, new int[]{1, 2}, false);
```

Comment:
How about a test where it fails if no barcode files are found?  How about a test if not all the tiles were found? My apologies if they are already there.